gensim/models/flsamodel.py

```diff
@@ -1434,11 +1320,12 @@ def __init__(
             min_count=1,
             window=5,
             vector_size=20,
-            workers=4,
         ):
 
         self.model = ...   # FIXME what is this?
         self.word_embedding = ...  # FIXME what is this?
+        #ERijck: this way users can do post-analysis. But storing them as an attribute
+        #       is not crucial.
```

Comment:
OK, but what does the `...` do?

It's Python's special syntax for ellipsis, or slicing, but I'm not familiar with using it for attribute values.

Reply:
It's a built-in ellipsis constant. People use it as a placeholder (TODO).

```
Python 3.10.6 (main, Nov 14 2022, 16:10:14) [GCC 11.3.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> x = ...
>>> x
Ellipsis
```

Reply:
Yes, that's what I wrote. But why is it here?

Reply:
If they're really "required" public attributes with deferred initialization, use simply `self.model = None`. No need for special ellipsis.

Plus maybe add a comment about why these attributes are not initialized right away. Unless it's obvious (sorry I haven't checked the code in detail).